toonz/sources/CMakeLists.txt

```diff
@@ -14,7 +14,11 @@ message("Thirdpary Library Search path:" ${THIRDPARTY_LIBS_HINTS})
 
 if (WIN32)
     message("Windows System")
-    set(QT_PATH "C:/Qt/Qt5.5.1/5.5/msvc2013_64" CACHE PATH "Qt instlattion directory")
+    if(CMAKE_SIZEOF_VOID_P EQUAL 4)
+        set(QT_PATH "C:/Qt/Qt5.5.1_32/5.5/msvc2013")
+    else()
+        set(QT_PATH "C:/Qt/Qt5.5.1/5.5/msvc2013_64")
+    endif()
```

Comment:
`CACHE` is needed for users who installed Qt to the other directory to change `QT_PATH`.